components/input/Input.cs

```diff
@@ -75,6 +75,8 @@ public class Input<TValue> : AntInputComponentBase<TValue>
 
         public Dictionary<string, object> Attributes { get; set; }
 
+        private string _inputValue;
```

Comment:
Should we use TValue instead of string ?

Reply:
You are right, I have fixed it.